src/components/capture/Create.tsx

```diff
@@ -23,6 +24,16 @@ import { getPathWithParam } from 'utils/misc-utils';
 
 const connectorType = 'capture';
 
+const trackEvent = (payload: any) => {
```

Comment:
Did you run into an issue narrowing the type of the `payload` argument?

Reply:
We just don't have the type created yet for the response of a subscription start.